src/main/java/org/session13/homework/librarymanagementsystem/librarymanagement/Book.java

```diff
@@ -0,0 +1,75 @@
+package org.session13.homework.librarymanagementsystem.librarymanagement;
+
+import org.session13.homework.librarymanagementsystem.profiles.Author;
+
+public class Book {
+
+    private String title;
+    private Author author;
+    private String ISBN;
+    private String publisher;
+    private int printedCopies;
+
+    public static class Builder {
+
+        private String title;
+        private Author author;
+        private String ISBN;
+        private String publisher;
+        private int printedCopies;
+
+        public Builder(String title, Author author) {
```

Comment:
We can skip for now this step, because we want just to do the OOP solution, but please keep in mind that when we user builders, we also need to ensure that incorect objects are not created. This is why we can use validations to be sure that we don't receive null/empty titles/authors).

Great job of using `builder`.